### PR TITLE
Screenshots R 2-4

### PR DESCRIPTION
**Pull request by dsarkar**
base branch darkar-release-2-4

- Android - missing R 2.4 screenshots
  - RAT
  - Event registration 
  - Internal Tracking ID: [EXPOSUREAPP-6946](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-6946) 

- iOS
  - Broken (both DE and EN versions)
iPhone 11 Pro-submissionflow_screenshot_test_certificate_entered_birthday
iPhone 11 Pro-traceLocation_006_createQRCodeInputScreen 
Internal Tracking ID: [EXPOSUREAPP-8004](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-8004) 
 
---
Internal Tracking ID: [EXPOSUREAPP-7325](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7325)

### CODE REVIEW

File: src/data/screenshots_de.json [hunk 1]
```diff
@@ -142,116 +156,139 @@
...
+                    { "filename": "/assets/screenshots/2-4/de/ios/iPhone 11 Pro-contact_journal_listing1_0001", "alt": "Kontakt-Tagebuch Bild 4 von 8" },
+                    { "filename": "/assets/screenshots/2-4/de/ios/iPhone 11 Pro-contact_journal_listing_add_persons", "alt": "Kontakt-Tagebuch Bild 5 von 8" },
+                    { "filename": "/assets/screenshots/2-4/de/ios/iPhone 11 Pro-contact_journal_listing_add_locations", "alt": "Kontakt-Tagebuch Bild 6 von 8" },
+                    { "filename": "/assets/screenshots/2-4/de/ios/iPhone 11 Pro-contact_journal_listing_edit_persons", "alt": "Kontakt-Tagebuch Bild 7 von 8" },
+                    { "filename": "/assets/screenshots/2-4/de/ios/iPhone 11 Pro-contact_journal_listing_edit_locations", "alt": "Kontakt-Tagebuch Bild 8 von 8" }
                 ]
             },
             {
-                "title": "Check-in",
+                "title": "Check-In - Scan QR-Code",
```
Comment (Ein-Tim) [inline src/data/screenshots_de.json:194]:
```suggestion
                "title": "Check-in - QR-Code scannen",
```

Why is this section only there for iOS?

File: src/data/screenshots_de.json [hunk 2]
```diff
@@ -142,116 +156,139 @@
...
+                    { "filename": "/assets/screenshots/2-4/de/ios/iPhone 11 Pro-event_checkin_004_mycheckins_emptyList", "alt": "Check In Bild 8 von 8" },
+                    { "filename": "/assets/screenshots/2-4/de/ios/iPhone 11 Pro-event_checkin_005_mycheckins_checkin", "alt": "Check In Bild 8 von 8" },
+                    { "filename": "/assets/screenshots/2-4/de/ios/iPhone 11 Pro-event_checkin_005_checkinDetails", "alt": "Check In Bild 8 von 8" },
+                    { "filename": "/assets/screenshots/2-4/de/ios/iPhone 11 Pro-event_checkin_007_checkinDetailsTime", "alt": "Check In Bild 8 von 8" },
+                    { "filename": "/assets/screenshots/2-4/de/ios/iPhone 11 Pro-event_checkin_006_mycheckins", "alt": "Check In Bild 8 von 8" },
+                    { "filename": "/assets/screenshots/2-4/de/ios/iPhone 11 Pro-event_checkin_008_checkinList", "alt": "Check In Bild 8 von 8" }
+                ]
+            },
+            {
+                "title": "Check-In - QR-Code erstellen",
```
Comment (Ein-Tim) [inline src/data/screenshots_de.json:206]:
```suggestion
                "title": "Check-in - QR-Code erstellen",
```

File: src/data/screenshots.json [hunk 3]
```diff
@@ -142,116 +156,139 @@
...
+                    { "filename": "/assets/screenshots/2-4/en/ios/iPhone 11 Pro-event_checkin_008_checkinList", "alt": "Check In Bild 8 von 8" }
+                ]
+            },
+            {
+                "title": "Check In - Create Event",
+                "anchor": "ios_check_in_create_event",
+                "images": [
+                    { "filename": "/assets/screenshots/2-4/en/ios/iPhone 11 Pro-traceLocation_004_emptyList", "alt": "Check In Bild 8 von 8" },
+                    { "filename": "/assets/screenshots/2-4/en/ios/iPhone 11 Pro-traceLocation_005_categories", "alt": "Check In Bild 8 von 8" },
+                    { "filename": "/assets/screenshots/2-4/en/ios/iPhone 11 Pro-traceLocation_006_createQRCodeInputScreen", "alt": "Check In Bild 8 von 8" },
```
Comment (Ein-Tim) [inline src/data/screenshots.json:211]: This screenshot is broken.

File: src/data/screenshots.json [hunk 4]
```diff
@@ -142,116 +156,139 @@
...
+                    { "filename": "/assets/screenshots/2-4/en/ios/iPhone 11 Pro-screenshot_second_health_certificate", "alt": "Vaccination Certficate Image 11 of 11" }
+                ]
+            },
+            {
+                "title": "Test Certficate",
+                "anchor": "ios_test_certficate",
+                "images": [
+                    { "filename": "/assets/screenshots/2-4/en/ios/iPhone 11 Pro-screenshot_certificate_overview_vaccination_blue", "alt": "Test Certficate Image" },
+                    { "filename": "/assets/screenshots/2-4/en/ios/iPhone 11 Pro-homescreenrisk_level_low_installation_14days_0001", "alt": "Test Certficate Image" },
+                    { "filename": "/assets/screenshots/2-4/en/ios/iPhone 11 Pro-submissionflow_screenshot_test_certificate_entered_birthday", "alt": "Test Certficate Image" },
```
Comment (Ein-Tim) [inline src/data/screenshots.json:289]: This screenshot is broken.

File: src/data/screenshots_de.json [hunk 5]
```diff
@@ -142,116 +156,139 @@
...
+                "anchor": "ios_check_in_create_event",
+                "images": [
+                    { "filename": "/assets/screenshots/2-4/de/ios/iPhone 11 Pro-traceLocation_004_emptyList", "alt": "Check In Bild 8 von 8" },
+                    { "filename": "/assets/screenshots/2-4/de/ios/iPhone 11 Pro-traceLocation_005_categories", "alt": "Check In Bild 8 von 8" },
+                    { "filename": "/assets/screenshots/2-4/de/ios/iPhone 11 Pro-traceLocation_006_createQRCodeInputScreen", "alt": "Check In Bild 8 von 8" },
```
Comment (Ein-Tim) [inline src/data/screenshots_de.json:211]: This screenshot is broken.

File: src/data/screenshots_de.json [hunk 6]
```diff
@@ -142,116 +156,139 @@
...
+                    { "filename": "/assets/screenshots/2-4/de/ios/iPhone 11 Pro-screenshot_second_health_certificate", "alt": "Vaccination Certficate Image 11 of 11" }
+                ]
+            },
+            {
+                "title": "Testzertifikat",
+                "anchor": "ios_test_certficate",
+                "images": [
+                    { "filename": "/assets/screenshots/2-4/de/ios/iPhone 11 Pro-screenshot_certificate_overview_vaccination_blue", "alt": "Test Certficate Image" },
+                    { "filename": "/assets/screenshots/2-4/de/ios/iPhone 11 Pro-homescreenrisk_level_low_installation_14days_0001", "alt": "Test Certficate Image" },
+                    { "filename": "/assets/screenshots/2-4/de/ios/iPhone 11 Pro-submissionflow_screenshot_test_certificate_entered_birthday", "alt": "Test Certficate Image" },
```
Comment (Ein-Tim) [inline src/data/screenshots_de.json:289]: This screenshot is broken.

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
The new screenshots are not well numbered anymore, e.g. the new check-in section calls all screenshots "Bild 8 von 8", so please fix these "alt"s 👍

File: src/data/screenshots_de.json (hunk 1 above)
Comment (dsarkar) [reply src/data/screenshots_de.json:194]: Hi, thanks. We don't have the corresponding screenshots for Android. Internal Tracking ID: EXPOSUREAPP-6946

File: src/data/screenshots.json (hunk 4 above)
Comment (dsarkar) [reply src/data/screenshots.json:289]: yes, I will create a bug ticket.

File: src/data/screenshots_de.json (hunk 5 above)
Comment (dsarkar) [reply src/data/screenshots_de.json:211]: same, will create a ticket

File: src/data/screenshots_de.json (hunk 6 above)
Comment (dsarkar) [reply src/data/screenshots_de.json:289]: ok, thanks

**Comment by dsarkar** [comment]
@Ein-Tim Thanks.

File: src/data/screenshots.json (hunk 3 above)
Comment (dsarkar) [reply src/data/screenshots.json:211]: ok

**Comment by Ein-Tim** [comment]
@dsarkar 

Thank you for creating the internal tickets, the broken screenshots should probably be replaced before this is merged 🙂

**Pull request merged**